sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,217 @@
+
+## Proposed Solution
+
+sBTC aims to mitigate Bitcoin’s limitations by combining the capability of the Stacks Blockchain with the reliability and security of Bitcoin. By enabling the secure movement of BTC in and out of the Stacks Blockchain via the sBTC protocol, users can interact with their BTC on Stacks using Clarity smart contracts and fast block times. The protocol is “secure” in that it is operated by a decentralized signer network, removing the risk of a single point of failure and trust in a single custodian. Users can deposit BTC into the protocol, seamlessly transact using sBTC on the Stacks blockchain, and have the freedom to redeem sBTC tokens for the underlying BTC at any time.
+
+### Programmability
+
+[Clarity](https://docs.stacks.co/clarity/overview) is the smart contract language on Stacks, which allows developers to encode essential business logic on a blockchain. Using smart contracts, developers can build more expressive decentralized applications that interact with sBTC, such as DeFi protocols, stablecoins, payments, and many others.
+
+### Fast Blocks
+
+The Stacks Nakamoto Upgrade, proposed in [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md#proposed-solution), enables fast blocks where user-submitted transactions will now take on the order of seconds, instead of tens of minutes. Thus, sBTC on Stacks Nakamoto will offer an improvement to Bitcoin’s current transaction times.
+
+The sBTC protocol not only addresses the limitations of the Bitcoin scripting system but also provides a secure and decentralized solution for utilizing Bitcoin in various applications.
+
+## Design
+
+While the first sBTC implementation is under development, the wrapped nature of the sBTC token means that any such system would have the following properties:
+
+- sBTC is a SIP-10 token backed 1:1 by BTC.
+- The sBTC peg wallet is maintained by the set of sBTC signers. These signers are responsible for the security and maintenance of the wallet, ensuring that sBTC is redeemable for BTC.
+- Bitcoin can be converted into sBTC within 3 Bitcoin blocks; and sBTC can be converted into Bitcoin within 6 Bitcoin blocks. sBTC relies on the forking behavior guaranteed by [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md) in order to maintain the peg wallet correctly across forks.
+
+## Specification
+
+Management of the sBTC peg wallet on the Bitcoin blockchain is decentralized, involving the sBTC Signer Set rather than a single custodian. At launch, the sBTC protocol will be maintained by 15 independent entities that make up the sBTC Signer Set. The eligibility criteria to become an sBTC Signer is determined through the community governance process of ratifying this SIP.
```

Comment:
I think the word "decentralize" is being twisted here to imply something that you don't mean.  The signer set is going to be chosen by this SIP, but the signer set is fundamentally closed-membership once this SIP is ratified, right?  Specifically, the signer set is going to be chosen by the sBTC working group, as mentioned later?

Reply:
i think agree with the author here - decentralized doesn't implicitly imply open membership, even though that may be how it's mostly understood today. 
a system may have closed membership and still be decentralized (assuming none of the closed members can manipulate a system)  - unless there's a definition i'm missing?

Reply:
I mean, the bigger problem is that "decentralized" is a very loaded buzzword with no technically-meaningful definition, so it's just as hard to promote its use as it is to discourage it.  However, the "vibes" I get is that "decentralized" implies "no distinguished authority decides how this works," which is _not_ true here.  This SIP is a distinguished authority that declares what the signer set shall be.

Reply:
that's a fair point. borrowing some terms from elsewhere in this document, what about a change like this?
```
Management of the sBTC peg wallet on the Bitcoin blockchain shall be managed by the proposed set of signers through a democratic process...
```

Reply:
d5b1ac0750e2d5c5690f87a8f5473ad755019545